research/README.MD

```diff
@@ -0,0 +1,41 @@
+# Research Directory
+Research directory will be the place to host various research work from community on Federated learning
+leveraging NVIDIA FLARE. **The code will not be maintained by NVIDIA FLARE team**, but will require Pull Request
+approval process. 
+
+## License
+By providing the code in NVFLARE repository, you will grant the research project in NVIDIA repo to be released under open source license
+Apache v2 License. 
+
+## Requirements
+Each research project should create a sub directory with following requirements
+
+* sub directory name must be in ASCII string and no longer than 35 characters long
+* Each project should include
+  * README.MD -- document must include
+    * objective 
+    * background
+    * description
+    * nvflare version used
```

Comment:
do we need detialed steps to run the code, including:
- data download and preparation 
- steps to run the code
- expected results